tests/integration_tests/test_lm/test_jsrun.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=protected-access, unused-argument, no-value-for-parameter
+import os
+import subprocess as sp
+
+import radical.utils as ru
+
+from unittest import mock, TestCase
+
+from radical.pilot.agent.launch_method.jsrun import JSRUN
+
+
+# ------------------------------------------------------------------------------
+#
+class TestTask(TestCase):
+
+    # --------------------------------------------------------------------------
+    #
+    @classmethod
+    def setUpClass(cls):
+        path = os.path.dirname(__file__) + '/../test_config/resources.json'
+        resources = ru.read_json(path)
+        hostname = 'summit'
+
+        for host in resources.keys():
+            if host in hostname:
+                cls.host = host
+                cls.resource = resources[host]
+                break
+
+    # --------------------------------------------------------------------------
+    #
+    @mock.patch.object(JSRUN, '__init__',   return_value=None)
+    @mock.patch('radical.utils.Logger')
+    def test_configure(self, mocked_init, mocked_Logger):
+        component = JSRUN(name=None, cfg=None, session=None)
+        component._log = mocked_Logger
+        component._configure()
+        self.assertEqual(component.launch_command, self.resource['jsrun_path'])
+
+    # --------------------------------------------------------------------------
+    #
+    @mock.patch.object(JSRUN, '__init__', return_value=None)
+    @mock.patch('radical.utils.Logger')
+    def test_command(self, mocked_init, mocked_Logger):
+        cfg = {
```

Comment:
this will be moved to a separate json file, right?

Reply:
Yes

Reply:
I created a task json file with the task and expected output. This is then used for executing the integration test.